modules/@apostrophecms/attachment/index.js

```diff
@@ -2,44 +2,13 @@ const _ = require('lodash');
 const path = require('path');
 const fs = require('fs');
 const Promise = require('bluebird');
-const uploadfs = require('uploadfs');
-const mkdirp = require('mkdirp');
 
 module.exports = {
   options: { alias: 'attachment' },
-  cascades: [ 'imageSizes' ],
-  imageSizes: {
-    add: {
-      max: {
-        width: 1600,
-        height: 1600
-      },
-      full: {
-        width: 1140,
-        height: 1140
-      },
-      'two-thirds': {
-        width: 760,
-        height: 760
-      },
-      'one-half': {
-        width: 570,
-        height: 700
-      },
-      'one-third': {
-        width: 380,
-        height: 700
-      },
-      'one-sixth': {
-        width: 190,
-        height: 350
-      }
-    }
-  },
 
   async init(self) {
     self.name = 'attachment';
-
+    self.imageSizes = self.apos.modules['@apostrophecms/uploadfs'].imageSizes;
```

Comment:
This seems like a deal breaker for an `uploadfs` module. Let's call it `uploader`.

Reply:
Thinking of documentation, I don't want people to have to think about what `uploadfs` is when they're adding image sizes.

Reply:
Since uploadfs exports its options, I can change this to `self.apos.uploadfs.options.imageSizes` to address this concern. Then there will be zero references to the module outside itself.

Reply:
I don't understand. So image sizes would be added to `attachments`?